app/models/computacenter/responsible_body_urns.rb

```diff
@@ -50,7 +50,7 @@ def computacenter_identifier
         end
       when 'FurtherEducationCollege'
         # potential n+1
-        schools.first&.ukprn
+        "FE#{schools.first&.ukprn}"
```

Comment:
```suggestion
        if schools.first&.ukprn
          "FE#{schools.first&.ukprn}"
        else
          ""
        end
```

I think we'll want to check for the UKPRN here so we don't just send `"FE"` - what do you think?